server/api/utils/notification_pusher.py

```diff
@@ -157,7 +157,9 @@ def _prepare_notification_args(
         notification_object: mlrun.common.schemas.Notification,
     ):
         message = (
-            f": {notification_object.message}" if notification_object.message else ""
+            f": {notification_object.message}"
+            if notification_object.message
+            else alert.summary
```

Comment:
Is there any chance the alert summary is also None?

Reply:
no, creating an alert without a summary would fail

Reply:
@Yacouby what if I created a notification not via alert? Since we are not validating the message anymore

Reply:
this flow is called only in the alerts flow.. there is another method with the same name as this one that is called in the NotificationPusher class, and there it is handled if the message is not supplied